Makefile

```diff
@@ -187,7 +187,12 @@ publish_mix: compile
 
 #==> Tests tasks
 
-test: test_erlang test_elixir
+test:
+	$(RM) -r .home
+	mkdir .home
+	HOME=$(CURDIR)/.home $(MAKE) test_erlang
```

Comment:
I could also set `$HOME` globally in the Makefile, and then make all test tasks depend on another task that actually creates `$HOME` if needed. I can give it a try if you agree.